.jshintrc

```diff
@@ -3,7 +3,7 @@
 	"curly": true,
 	"eqeqeq": true,
 	"eqnull": true,
-	"esversion": 3,
+	"esversion": 8,
```

Comment:
Is it possible to leave the ESLint version changes out of this PR? Just because it seems unrelated and may have implications for other PRs which are hard to foresee. This also applies (maybe even more) to the newly added `.eslintrc.js` file above.

Reply:
It's related because the ECMAScript version for `src/js/_enqueues/lib/emoji-loader.js` is no longer ES3.